requests_toolbelt/adapters/appengine.py

```diff
@@ -59,6 +78,22 @@ def init_poolmanager(self, connections, maxsize, block=False):
         self.poolmanager = _AppEnginePoolManager(self._validate_certificate)
 
 
+class AppEngineInsecureAdapter(adapters.HTTPAdapter):
```

Comment:
Better question, why did you copy an entire class definition instead of doing

```py
class InsecureAppEngineAdapter(AppEngineAdapter):

    def __init__(self, validate_certificate=False, *args, **kwargs):
        super(InsecureAppEngineAdapter, self).__init__(validate_certificate, *args, **kwargs)
```

The AppEngineAdapter already creates the `_AppEnginePoolManager`.